provider/testdata/data/cldr/cldr-cal-japanese-full/main/sr-Cyrl/ca-japanese.json

```diff
@@ -0,0 +1,1248 @@
+                "171": "Шочо (1428–1429)",
+                "172": "Еикјо (1429–1441)",
+                "173": "Какитсу (1441–1444)",
+                "174": "Бун-ан (1444-1449)",
+                "175": "Хотоку (1449–1452)",
+                "176": "Кјотоку (1452–1455)",
+                "177": "Кошо (1455–1457)",
+                "178": "Чороку (1457–1460)",
+                "179": "Каншо (1460–1466)",
+                "180": "Буншо (1466–1467)",
+                "181": "Онин (1467–1469)",
+                "182": "Бунмеи (1469–1487)",
+                "183": "Чокјо (1487–1489)",
+                "184": "Ентоку (1489–1492)",
+                "185": "Меио (1492–1501)",
+                "186": "Бунки (1501–1504)",
+                "187": "Еишо (1504–1521)",
+                "188": "Таиеи (1521–1528)",
+                "189": "Кјороку (1528–1532)",
+                "190": "Тенмон (1532–1555)",
+                "191": "Кођи (1555–1558)",
+                "192": "Еироку (1558–1570)",
+                "193": "Генки (1570–1573)",
+                "194": "Теншо (1573–1592)",
+                "195": "Бунроку (1592–1596)",
+                "196": "Кеичо (1596–1615)",
+                "197": "Генва (1615–1624)",
+                "198": "Кан-еи (1624-1644)",
+                "199": "Шохо (1644–1648)",
+                "200": "Кеиан (1648–1652)",
+                "201": "Шу (1652–1655)",
+                "202": "Меирјаку (1655–1658)",
+                "203": "Мањи (1658–1661)",
+                "204": "Канбун (1661–1673)",
+                "205": "Енпо (1673–1681)",
+                "206": "Тенва (1681–1684)",
+                "207": "Јокјо (1684–1688)",
+                "208": "Генроку (1688–1704)",
+                "209": "Хоеи (1704–1711)",
+                "210": "Шотоку (1711–1716)",
+                "211": "Кјохо (1716–1736)",
+                "212": "Генбун (1736–1741)",
+                "213": "Канпо (1741–1744)",
+                "214": "Енкјо (1744–1748)",
+                "215": "Кан-ен (1748-1751)",
+                "216": "Хорјаку (1751–1764)",
+                "217": "Меива (1764–1772)",
+                "218": "Ан-еи (1772-1781)",
+                "219": "Тенмеи (1781–1789)",
+                "220": "Кансеи (1789–1801)",
+                "221": "Кјова (1801–1804)",
+                "222": "Бунка (1804–1818)",
+                "223": "Бунсеи (1818–1830)",
+                "224": "Тенпо (1830–1844)",
+                "225": "Кока (1844–1848)",
+                "226": "Каеи (1848–1854)",
+                "227": "Ансеи (1854–1860)",
+                "228": "Ман-ен (1860-1861)",
+                "229": "Бункју (1861–1864)",
+                "230": "Генђи (1864–1865)",
+                "231": "Кеико (1865–1868)",
+                "232": "Меиђи",
+                "233": "Таишо",
+                "234": "Шова",
+                "235": "Хаисеи",
+                "236": "Реива"
+              },
+              "eraNarrow": {
```

Comment:
`question`: is there a value in storing it as a hashmap rather than array?

Reply:
We don't have zero copy hashmaps, I'm using LiteMap since that can be easily upgraded to ZeroMap

Reply:
why not `zerovec`?

Reply:
I mean, we need a map from era dates to era names that can be efficiently looked up.

But also: are you sure you're looking at the right part of the code? This is the cldr-json dump, we have no control over this format.

Reply:
Note that this is a CLDR JSON file, not an ICU4X file.  We will process this into something better, which I expect will be in a follow-up PR.  (This is part of the reason why I suggested that we not add these files until we have the transformer code ready)

Reply:
All I see is:

```json
{
  "1": "Foo",
  "2": "Bar",
  "3": "Baz"
}
```

which seem like it could be:

```json
[
  "Foo",
  "Bar",
  "Baz"
]
```

Reply:
Right, that's the CLDR data format, from upstream.

Note that the keys here can also be things like `"1-variant"` etc (check out ca-gregorian.json for this). We currently ignore these but ultimately we also don't use the numerical mappings _anyway_.

Reply:
ah, gotcha. That makes sense. thanks!